src/magento-payments/payouts.md

```diff
@@ -0,0 +1,128 @@
+---
+ee_only: false
+title: Payouts
+---
+
+Payouts shows comprehensive payout information at-a-glance, allowing you full transparency into the processed volume, payment balance, and detailed reporting on the transaction level for financial reconciliation.
+
+You don't have to open multiple dashboards or views to cross-reference orders and payments or reconcile accounts. Magento Payments enables you to take all these actions from one place---Payouts---so that you can view and manage your payouts efficiently.
+
+See payments across multiple providers, as well as the linked Magento order and transaction IDs, transaction amounts, payment method per transaction, and more, all within Payouts in the Admin.
```

Comment:
We're not onboarding any other provider except PayPal for GA.
This mention about multiple providers may create further questions.
We should get someone from the Product team to review this please.

Reply:
Good point @h-madan